build-system/tasks/prepend-global/index.js

```diff
@@ -26,6 +26,7 @@ const util = require('util');
 const exec = util.promisify(childProcess.exec);
 
 const {red, cyan} = colors;
+const customConfigFile = 'build-system/global-configs/custom-config.json';
```

Comment:
Could you add a comment above this line with a link that points to https://github.com/ampproject/amphtml/blob/master/build-system/global-configs/README.md#custom-configjson?

Reply:
Done.